server/src/main/java/nl/inl/blacklab/server/search/ResultsCache.java

```diff
@@ -90,7 +90,7 @@ public T get(long timeout, TimeUnit unit) throws InterruptedException, Execution
         }
 
         @Override
-        public T peek() throws ExecutionException {
+        public T peek() {
```

Comment:
This class has changed however this change won't conflict with the new version

Reply:
No problem, I merged those changes.